### DISCUSSION TITLE
Taking screenshots without color profiles, or as SRGB?

### DISCUSSION THREAD

**Discussion by YousufSSyed** [iina/iina#4751] [Q&A] [answered]
I take screenshots and edit them, but first I have to change the color space to SRGB or otherwise I have issue. I want the screenshots to be taken as SRGB instead of profiles like "Rec. ITU-R BT.709-5"

**Comment by low-batt** [comment, marked as answer]
For audio/video support IINA uses a library from the [mpv](https://github.com/mpv-player/mpv) project. It is `mpv` that generates the screenshot.

The `mpv` manual entry for the [screenshot-tag-colorspace](https://mpv.io/manual/stable/#options-screenshot-tag-colorspace) option says:
> **--screenshot-tag-colorspace=<yes|no>**
>    Tag screenshots with the appropriate colorspace (default: yes).
>
>    Note that not all formats support this. When it is unsupported, or when this option is disabled, screenshots will be converted to sRGB before being written.

Possibly setting this option to `no` will do what you want. _However_ there currently is a gotcha with doing this. IINA will set this option to `yes` when HDR mode is enabled stepping on any user provided setting.

Follow these instructions to test setting this `mpv` option:

- Start IINA
- Click on  `Settings…` under the `IINA` menu
- The settings panel appears
- On the left side of the panel click on `Advanced`
- Slide the `Enable advanced settings` toggle button to be on (blue)
- In the `Additional mpv options` section click on `+`
- A new entry appears in the table
- Double click on `name` and replace it with `screenshot-tag-colorspace`
- Double click on `value` and replace it with `no`
- **Restart IINA** to activate the new settings

See if that works.

Please enter a feature request issue asking for IINA to provide the ability to control this in IINA's settings.

**Comment by hooke007** [comment]
You also need to make sure iina's ICC is diabled.
